Src/Support/GoogleApis.Core/Apis/Json/NewtonsoftJsonSerializer.cs

```diff
@@ -75,10 +76,13 @@ public static NewtonsoftJsonSerializer Instance
         static NewtonsoftJsonSerializer()
         {
             // Initialize the Newtonsoft serializer.
-            JsonSerializerSettings settings = new JsonSerializerSettings();
-            settings.NullValueHandling = NullValueHandling.Ignore;
-            settings.Converters.Add(new RFC3339DateTimeConverter());
-            newtonsoftSerializer = JsonSerializer.Create(settings);
+            s_settings = new JsonSerializerSettings

```

Comment:
Okay - I'm not entirely sure I agree, as the code within the generated library itself wasn't a problem (unless I've missed something) but I don't mind enough to try to convince you :)